app/build.gradle.kts

```diff
@@ -98,4 +98,6 @@ dependencies {
     implementation(libs.colorpicker)
 
     implementation(libs.autofill)
-}
\ No newline at end of file
+    implementation(libs.template)
+    implementation(libs.slf4j.android)
```

Comment:
Actually you don't need to implement the dependency R8 did throw on. You can just add like it said `-dontwarn org.slf4j.impl.StaticLoggerBinder` to `app/proguard-rules.pro`.

Reply:
Yes but then the engine wouldn't log. I don't think it's desirable.

Reply:
Okay fair enough :+1: